masonry/src/convert_winit_event.rs

```diff
@@ -318,316 +321,320 @@ pub(crate) fn winit_physical_key_to_kbt_code(physical_key: WinitPhysicalKey) ->
 pub(crate) fn winit_key_to_kbt_key(winit_key: &WinitKey) -> Key {
     match winit_key {
         WinitKey::Character(c) => Key::Character(c.to_string()),
-        WinitKey::Unidentified(_) => Key::Unidentified,
-        WinitKey::Dead(_) => Key::Dead,
+        WinitKey::Unidentified(_) => Key::Named(NamedKey::Unidentified),
+        WinitKey::Dead(_) => Key::Named(NamedKey::Dead),
         WinitKey::Named(named_key) => match named_key {
-            WinitNamedKey::Alt => Key::Alt,
-            WinitNamedKey::AltGraph => Key::AltGraph,
-            WinitNamedKey::CapsLock => Key::CapsLock,
-            WinitNamedKey::Control => Key::Control,
-            WinitNamedKey::Fn => Key::Fn,
-            WinitNamedKey::FnLock => Key::FnLock,
-            WinitNamedKey::NumLock => Key::NumLock,
-            WinitNamedKey::ScrollLock => Key::ScrollLock,
-            WinitNamedKey::Shift => Key::Shift,
-            WinitNamedKey::Symbol => Key::Symbol,
-            WinitNamedKey::SymbolLock => Key::SymbolLock,
-            WinitNamedKey::Meta => Key::Meta,
-            WinitNamedKey::Hyper => Key::Hyper,
-            WinitNamedKey::Super => Key::Super,
-            WinitNamedKey::Enter => Key::Enter,
-            WinitNamedKey::Tab => Key::Tab,
+            WinitNamedKey::Alt => Key::Named(NamedKey::Alt),
+            WinitNamedKey::AltGraph => Key::Named(NamedKey::AltGraph),
+            WinitNamedKey::CapsLock => Key::Named(NamedKey::CapsLock),
+            WinitNamedKey::Control => Key::Named(NamedKey::Control),
+            WinitNamedKey::Fn => Key::Named(NamedKey::Fn),
+            WinitNamedKey::FnLock => Key::Named(NamedKey::FnLock),
+            WinitNamedKey::NumLock => Key::Named(NamedKey::NumLock),
+            WinitNamedKey::ScrollLock => Key::Named(NamedKey::ScrollLock),
+            WinitNamedKey::Shift => Key::Named(NamedKey::Shift),
+            WinitNamedKey::Symbol => Key::Named(NamedKey::Symbol),
+            WinitNamedKey::SymbolLock => Key::Named(NamedKey::SymbolLock),
+            WinitNamedKey::Meta => Key::Named(NamedKey::Meta),
+            #[expect(deprecated, reason = "Map even the deprecated keys.")]
+            WinitNamedKey::Hyper => Key::Named(NamedKey::Hyper),
+            #[expect(deprecated, reason = "Map even the deprecated keys.")]
+            WinitNamedKey::Super => Key::Named(NamedKey::Super),
```

Comment:
This part is wrong probably? Should be:
```suggestion
            WinitNamedKey::Meta => Key::Named(NamedKey::Super),
            #[expect(deprecated, reason = "Map even the deprecated keys.")]
            WinitNamedKey::Hyper => Key::Named(NamedKey::Hyper),
            #[expect(deprecated, reason = "Map even the deprecated keys.")]
            WinitNamedKey::Super => Key::Named(NamedKey::Meta),
```

(If you want to match Winit `v0.31` that is)

Reply:
But we're still using winit 0.30 ... so I'm not sure. Once 0.31 hits and we update to that ... well, then this won't be needed ideally since we'd both be using `keyboard-types`.

Reply:
I'm not that familiar with how Xilem handles keyboard input, but I guess, it kinda depends on how you want your users to experience things? I think you'd want them to use `keyboard_types::NamedKey::Meta` in their code, right?